docs/SUMMARY.md

```diff
@@ -39,7 +39,16 @@
     * [Excluding patterns](earthfile/earthignore.md)
 * [The `earthly` command](earthly-command/earthly-command.md)
 * [Configuration reference](earthly-config/earthly-config.md)
-* [CI integration](./ci-integration.md)
+* [CI integration](ci-integration/ci-integration.md)
+  * [Building An Image](ci-integration/building-an-image.md)
+  * [Pull-Through Cache](ci-integration/pull-through-cache.md)
+  * [Remote Buildkit](ci-integration/remote-buildkit.md)
+  * Guides
```

Comment:
Wondering if we can use a different word for this subsection, given that we already have "Guides" as a section somewhere else?

Reply:
"Vendor-Specific Guides" is what I had this as before?